theano/tensor/tests/test_nlinalg.py

```diff
@@ -516,3 +518,29 @@ def test_numpy_compare(self):
             t_n = f(A[2][i])
             n_n = numpy.linalg.norm(A[2][i], A[3][i])
             assert _allclose(n_n, t_n)
+
+
+class TestLogAbsDet(unittest.TestCase):
+
+    def setUp(self):
+        utt.seed_rng()
+        self.op_class = LogAbsDet
+        self.op = logabsdet
+
+    def validate(self, input_mat):
+        x = theano.tensor.matrix()
+        f = theano.function([x], self.op(x))
+        out = f(input_mat)
+        svd_diag = numpy.linalg.svd(input_mat, compute_uv=False)
+        numpy_out = numpy.sum(numpy.log(numpy.abs(svd_diag)))
```

Comment:
That would be the slogdet function then, but it probably does something a bit different since the results are differ (relatively) by about 1E-8, as I discussed in the original issue (here)[https://github.com/Theano/Theano/issues/150]. Maybe I should compare against the slogdet but increase the tolerance of the allclose?